packages/dcos-diagnostics/buildinfo.json

```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-diagnostics.git",
-    "ref": "2f0d206619abbb083b6ac2b43efadf884a24769c",
+    "ref": "3734e2e03644449500427fb916289c4007dc5106",
```

Comment:
would you mind making this `1389e1499b0790a9bdf041514fe173a395a4b7b8` so my change is also in there?